draft-ietf-quic-manageability.md

```diff
@@ -767,12 +781,12 @@ signal ({{sec-teardown}}) means that this state must be purged either through
 timers or through least-recently-used eviction, depending on application
 requirements.
 
-{{?RFC4787}} recommends a 2 minute timeout interval for UDP, however,
-often timer are lower in the range of 15 to 30 second. In constrast
-{{?RFC5382}} recommends a timeout of more than 2 hours for TCP, given TCP is
-a connection-oriented protocol with well defined closure semantics. For network
-devices that are QUIC-aware, it is recommended to also use
-longer timeouts for QUIC traffic, as QUIC is connection-oriented and as such
+{{?RFC4787}} recommends a 2 minute timeout interval for UDP. However,
+timers are often lower, in the range of 15 to 30 seconds. In contrast,
```

Comment:
```suggestion
timers can be lower, in the range of 15 to 30 seconds. In contrast,
```

Reply:
I think "often" is correct here but fine...